### DISCUSSION TITLE
fail to download pigsty source from CDN

### DISCUSSION THREAD

**Discussion by wuxiao-ling197** [pgsty/pigsty#345] [Q&A]
![微信截图_20231129172113](https://github.com/Vonng/pigsty/assets/77096395/e2bb2804-6ead-408d-9ff3-aea0f3de71d2)

environment : Rocky8.9

when I used a normal user to install , it failed , but if I changed to root , it succeded , but it`s obviously I can`t access  "./pigsty" menu by normal user , how could I solve this problem ？

**Comment by Vonng** [comment]
well if you can not download from the CDN, you can just download from Github or other place and follow the installation instructuions: https://doc.pigsty.cc/#/INSTALL?id=download
